components/view_image/view_image.jsx

```diff
@@ -26,7 +26,12 @@ export default class ViewImageModal extends React.PureComponent {
         /**
          * The post the files are attached to
          */
-        post: PropTypes.object.isRequired,
+        postId: PropTypes.string,
```

Comment:
we should probably have custom types for this kind of thing, to prevent people comparing a post.id with a channel.name (or a channel.id) and notice that on compile time

Reply:
I agree!